goal_src/jak2/engine/gfx/texture/texture-anim.gc

```diff
@@ -774,18 +799,41 @@ struct SlimeInput {
                                   bucket
                                   )
        (pc-texture-anim-flag start-anim-array dma-buf)
-       (pc-texture-anim-flag clouds-and-fog dma-buf :qwc 4)
+       (pc-texture-anim-flag clouds-and-fog dma-buf :qwc (/ (psize-of sky-input) 16))
        (make-sky-input (the sky-input (-> dma-buf base)))
-       (&+! (-> dma-buf base) 64)
+       (&+! (-> dma-buf base) (psize-of sky-input))
        (pc-texture-anim-flag finish-anim-array dma-buf)
-       (dotimes (i 8) ;; intentially skipping fog here!!
+       (dotimes (i 9) ;; intentially skipping fog here!!
```

Comment:
This should actually be 8, and skip the last one (idx 8). The last one is the fog animation.

The code below this sets `anim-idx` to 8, then doesn't return, so the loop below updates the fog animation.

Reply:
oops, definitely didn't mean to change that.